tests/sam/env.json.template

```diff
@@ -0,0 +1,5 @@
+{
+  "BagitValidatorFunction": {
```

Comment:
This should be updated to `S3BagitValidatorFunction`. When I updated the `tests/sam/env.json`, I no longer received the error noted in my comment on line 54 of `tests/sam/README.md`. 🤓

Reply:
Ah, good catch!  Thanks.